### ISSUE TITLE
How control plane/worker nodes identified by the cluster

### ISSUE THREAD

**Issue by vinusorout** [kelseyhightower/kubernetes-the-hard-way#639]
Hello,

Lets say I have 4 control plane in a subnet, and I want three of them to connect to one cluster and one to another cluster.
Similarly with 4 worker nodes in same subnet, I want three to connect to one cluster and one to other cluster.

I know it will be a bad design but I just want to understand how cluster consider a node its control plane/worker node, Is it governed by the certificates? Please provide some details if possible!

**Comment by vinusorout** [comment]
If I am not wrong it is identified by the .kubeconfig created for kubelets and kube-apiserver file “—server”option?

**Comment by kelseyhightower** [comment]
Yes, the Kubelet will register with the default api server in the kubeconfig.
